src/tests/PhysicsTest.cpp

```diff
@@ -408,3 +408,52 @@ TEST_F(PhysicsManagerTest, TestVelocityControl) {
     ASSERT_EQ(physicsManager_->getAngularVelocity(objectId), Magnum::Vector3{});
   }
 }
+
+TEST_F(PhysicsManagerTest, TestSceneNodeAttachment) {
+  // test attaching/detaching existing SceneNode to/from physical simulation
+  LOG(INFO) << "Starting physics test: TestSceneNodeAttachment";
+
+  std::string objectFile = Cr::Utility::Directory::join(
+      dataDir, "test_assets/objects/transform_box.glb");
+
+  std::string sceneFile =
+      Cr::Utility::Directory::join(dataDir, "test_assets/scenes/plane.glb");
+
+  initScene(sceneFile);
+
+  esp::assets::PhysicsObjectAttributes physicsObjectAttributes;
+  physicsObjectAttributes.setString("renderMeshHandle", objectFile);
+  resourceManager_.loadObject(physicsObjectAttributes, objectFile);
```

Comment:
Underscore is not needed.

Reply:
This is a call to `PhysicsManagerTest::resourceManager_`, so underscore is needed. Did I miss your meaning?

Reply:
Oh, I see. This is the member variable of the class PhysicsManagerTest.

My bad.